### DISCUSSION TITLE
Why credit card accounts don't have a source account?

### DISCUSSION THREAD

**Discussion by killkrt** [firefly-iii/firefly-iii#6817] [Firefly III] [answered]
I am new FireFly III user and I think it's a great software and it is also very easy to install (I used Docker), so kudos to @JC5 for the great job that he is being making!

Anyway I struggle to understand how credit card accounts work. I've notice that in credit card option there is an option called "Credit card monthly payment date", so I was expecting also to have an option for setting the source account used for making a transfer at that date (each month).

So I cannot understand if I missing something or if I am supposed to create manually a transfer and that's option it is only used as reminder.

**Comment by JC5** [comment, marked as answer]
I added it to see if I could implement something like that but as it turns out, this works so differently for everybody that I never really bothered implementing anything. For the time being, it's cosmetic only.

**Reply by killkrt** [reply]
Ah, ok... Thanks for the clarification!
